### PR TITLE
Fix resume error

### PR DESCRIPTION
**Pull request by PietroPasotti**
On sleep/resume, the tempo charm goes into error because it receives a custom notice event for a service that doesn't exist in pebble (yet)?
Unclear what's causing this, but it's a pain.
Workaround and test included.

### CODE REVIEW

**Review by mmkay — APPROVED** [review]

File: tests/scenario/test_charm.py [hunk 1]
```diff
@@ -179,3 +184,22 @@ def test_tracing_storage_is_configured_to_s3_if_s3_relation_filled(
...
+    container, tempo = _tempo_mock_with_initial_config(tmp_path, tempo_ready_svc_exists=False)
+
+    state = State(leader=True, containers=[tempo])
+
+    # WHEN we receive a custom-notice event
+    with caplog.at_level("DEBUG"):
+        os.environ["SCENARIO_SKIP_CONSISTENCY_CHECKS"] = "1"
+        # scenario doesn't play nice in this very edge case
+        context.run(_BoundNotice(Notice(Tempo.tempo_ready_notice_key), tempo).event, state)
+        del os.environ["SCENARIO_SKIP_CONSISTENCY_CHECKS"]
```
Comment (dimaqq) [inline tests/scenario/test_charm.py:200]: Minor: consider https://docs.pytest.org/en/stable/reference/reference.html#pytest.MonkeyPatch.setenv instead

Comment (PietroPasotti) [reply tests/scenario/test_charm.py:200]: uh, cool, didn't know about that
